src/flint/gfp_elem.jl

```diff
@@ -469,6 +469,11 @@ function (R::fpField)(a::Union{fpFieldElem, zzModRingElem, FpFieldElem, ZZModRin
    end
 end
 
+function (R::fpField)(a::Vector{<:IntegerUnion})
+   isone(length(a)) || error("Coercion impossible")
```

Comment:
Any particular reason why you do not  allow length zero? You allow it in the `fqPolyRepField` case.

Reply:
Ah this was a simple oversight.  Fixed for now (unless Claus wants to require `length(a) == degree(R)`)